listenbrainz/webserver/views/test/test_recommendations_cf_recording.py

```diff
@@ -0,0 +1,145 @@
+        user = _get_user('vansika')
+        recommendations_cf_recording._get_template(active_section='top_artist', user=user)
+        self.assertTemplateUsed('recommendations_cf_recording/top_artist.html')
+        self.assert_context('active_section', 'top_artist')
+        self.assert_context('user', user)
+        error_msg="Recommended tracks for the user have not been calculated. Check back later."
+        self.assert_context('error_msg', error_msg)
+
+        user = _get_user('vansika')
+        recommendations_cf_recording._get_template(active_section='similar_artist', user=user)
+        self.assertTemplateUsed('recommendations_cf_recording/similar_artist.html')
+        self.assert_context('active_section', 'similar_artist')
+        self.assert_context('user', user)
+        error_msg="Recommended tracks for the user have not been calculated. Check back later."
+        self.assert_context('error_msg', error_msg)
+
+    def test_get_template_missing_rec_top_artist(self):
+        user = _get_user('vansika_2')
+        recommendations_cf_recording._get_template(active_section='top_artist', user=user)
+        self.assertTemplateUsed('recommendations_cf_recording/top_artist.html')
+        self.assert_context('active_section', 'top_artist')
+        self.assert_context('user', user)
+        error_msg="Looks like you weren't active last week. Check back later."
+        self.assert_context('error_msg', error_msg)
+
+    def test_get_template_missing_rec_top_artist(self):
+        user = _get_user('vansika_1')
+        recommendations_cf_recording._get_template(active_section='similar_artist', user=user)
+        self.assertTemplateUsed('recommendations_cf_recording/similar_artist.html')
+        self.assert_context('active_section', 'similar_artist')
+        self.assert_context('user', user)
+        error_msg="Looks like you weren't active last week. Check back later."
+        self.assert_context('error_msg', error_msg)
+
+    @patch('listenbrainz.db.recommendations_cf_recording.get_user_recommendation')
+    @patch('listenbrainz.webserver.views.recommendations_cf_recording._get_listens_from_recording_mbid')
+    def test_get_template(self, mock_get_listens, mock_get_rec):
+        self.temporary_login(self.user['login_id'])
+        user = _get_user('vansika_1')
+
+        mock_get_listens.return_value = [{
+            'listened_at' : 0,
+            'track_metadata' : {
+                'artist_name' : "Ultravox",
+                'track_name' : "Serenade (special remix)",
+                'release_name' : "Quartet",
+                'additional_info' : {
+                    'recording_mbid' : "af5a56f4-1f83-4681-b319-70a734d0d047",
+                    'artist_mbids' : ["6a70b322-9aa9-41b3-9dce-824733633a1c"]
+                }
+            },
+            'score': 0.756
+        }]
+        recommendations_cf_recording._get_template(active_section='top_artist', user=user)
```

Comment:
@paramsingh test fail here saying current_user doesn't exist. Can you have a look?